utreexo-validation-bip.md

```diff
@@ -160,20 +160,19 @@ cannot be proven with the Utreexo accumulator (and SPV proofs already provide th
 For these reasons, outputs which are spent in the same block where they are created 
 are omitted from the accumulator, and those inputs are omitted from block proofs.
 
-#### Order of operations
+## Order of operations
 The Utreexo accumulator lacks associative properties during addition and the
 ordering of which UTXO hash gets added first is consensus critical. For
-the modification of the accumulator the steps are as follows:
+the modification of the accumulator, the steps are as follows:
 
-1: Batch remove the UTXOs that were spent in the block based on the algorithm
-   defined in bip ???.  Deletions are order-independent.
-2: Batch add all non-excluded outputs in the order they're included in the
-   Bitcoin block.
+1. Verify that the UTXOs that were spent in the block. Raise error if the verification fails.
```

Comment:
This phrase sounds weird. It tells us to validate the scripts for transactions spending those outputs?